contracts/storage/ExchangeStorage.sol

```diff
@@ -24,3 +24,9 @@ abstract contract ExchangeStorageV1 {
     // value: the last timestamp when a trader exceeds price limit when closing a position/being liquidated
```

Comment:
// _lastOverPriceLimitTimestampMap is deprecated

Reply:
fixed in [cc5216e](https://github.com/perpetual-protocol/perp-curie-contract/pull/63/commits/cc5216e184de016401e236049633e8fed03710c7)